src/graph/resolvers/campaign.js

```diff
@@ -20,6 +20,12 @@ const getNotifyDefaults = async (advertiserId, user) => {
   return notify;
 };
 
+const updateNotifications = async (campaignId) => {
+  contactNotifier.scheduleCampaignCreated({ campaignId });
```

Comment:
The `updateNotifications` isn't awaiting the scheduled email calls, is that intentional?

Reply:
Yes